openedx/stanford/common/djangoapps/enrollment/views.py

```diff
@@ -166,8 +169,11 @@ def delete(self, request, course_id):
         if email_students:
             course = get_course_by_id(course_key)
             email_params = get_email_params(course, auto_enroll)
-            if User.objects.filter(email=email).exists():
+            try:
                 user = User.objects.get(email=email)
+            except User.DoesNotExist:
+                pass
+            else:
```

Comment:
Since we're performing the same lookup twice (identical code) and we're actually only using the lookup to grab the language (and not anything else from the `user` object), we should just share the code via a helper:

```python
def lookup_language(email):
    try:
        user = User.objects.get(email=email)
    except User.DoesNotExist:
        language = None
    else:
        language = get_user_email_language(user)
    return language

# and then used like:
if email_students:
    language = lookup_language(email)
    # ...
```